tla-types/src/main/scala/at/forsyte/apalache/tla/typecomp/subbuilder/ActionBuilder.scala

```diff
@@ -0,0 +1,32 @@
+package at.forsyte.apalache.tla.typecomp.subbuilder
+
+import at.forsyte.apalache.tla.typecomp.TBuilderInstruction
+import at.forsyte.apalache.tla.typecomp.unsafe.UnsafeActionBuilder
+import at.forsyte.apalache.tla.typecomp.BuilderUtil.binaryFromUnsafe
+
+/**
+ * Builder for TlaActionOper expressions.
```

Comment:
```suggestion
 * Type-safe for TlaActionOper expressions.
```

Reply:
Type-safe builder for TlaActionOper :)

Reply:
We have this same doc convention ("Builder" vs "Unsafe builder") in all the other sub-builders, why should this one be different?

Reply:
Then I overlooked it on the other ones.
I think it's better to be specific.